configure.ac

```diff
@@ -2102,6 +2102,51 @@ esac
 AC_SUBST(LDAPLIB)
 AC_SUBST(LBERLIB)
 
+AC_ARG_WITH(systemd,
+  AS_HELP_STRING([--without-systemd],
+                 [Do not use systemd API to send start-up completion
+                  notification. Default: auto-detect]), [
+case "$with_systemd" in
+  yes|no)
+    : # Nothing special to do here
+    ;;
+  *)
+    if test ! -d "$withval" ; then
+      AC_MSG_ERROR([--with-systemd path does not point to a directory])
+    fi
+    SYSTEMD_PATH="-L$with_systemd/lib"
+    CPPFLAGS="-I$with_systemd/include $CPPFLAGS"
+  esac
+])
+AH_TEMPLATE(USE_SYSTEMD,[systemd support is available])
+if test "x$with_systemd" != "xno" -a "x$squid_host_os" = "xlinux"; then
```

Comment:
This is not my area of expertise so please feel free to dismiss the following question if the answer is obvious to an expert: Why do we need to exclude non-Linux environments? In other words, should not the libsystemd check drive the decision here rather than the spelling of the host OS name? I understand that systemd may be specific to Linux-like OSes (today), but that does not seem like a good enough reason -- many OS features are or were OS-specific, but if we can directly test the feature support itself, should not we?

Reply:
No strong reason. To not perturb BSD folks with (for now) useless libraries checks? hehe

Reply:
OK, I would remove this OS spelling check in this case, but it is your call. I am marking this change request as addressed.